docs/excel/excel-add-ins-ranges-large.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Read or write to large ranges using the Excel JavaScript API
+description: 'Learn how to read or write to large ranges with the Excel JavaScript API.'
+ms.date: 04/02/2021
+localization_priority: Normal
+---
+
+# Read or write to a large range using the Excel JavaScript API
+
+This article describes how to handle reading and writing to large ranges with the Excel JavaScript API. For the complete list of properties and methods that the `Range` object supports, see the [Excel.Range class](/javascript/api/excel/excel.range).
```

Comment:
Since this article isn't about a subset of Range operations, but rather a platform issue, let's drop the Range object link.
```suggestion
This article describes how to handle reading and writing to large ranges with the Excel JavaScript API.
```